src/app/modules/donaciones/donaciones-routing.module.ts

```diff
@@ -0,0 +1,20 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { ListarDonacionesComponent } from './pages/listar-donaciones/listar-donaciones.component';
+
+const routes: Routes = [
+  {
+    path: 'listarDonaciones',
```

Comment:
better to use just `''` since you're setting `path: 'donaciones'` at app-routing.